tests/test_ipython_magics.py

```diff
@@ -45,12 +45,14 @@ def _validate_magics_with_black(before: Sequence[str], after: Sequence[str]) ->
     diff = difflib.unified_diff(before, after)
     result = "".join(i for i in diff if any([i.startswith("+ "), i.startswith("- ")]))
     expected = (
+        '-    "\\n",\n'
```

Comment:
this happens from removing the newline after the magic. need to decide whether to standardise, or whether this is ok. maybe it's OK